AI Features: add AI Skills topic (#8830)
DevExpress/devextreme-documentation: concepts/90 AI Features/20 DevExpress MCP Server Configuration/00 DevExpress MCP Server Configuration.md

> The DevExpress MCP server connects MCP-compatible AI tools with the DevExpress documentation library, including DevExtreme documentation. The MCP server ensures AI tools such as GitHub Copilot have direct access to the latest information on DevExtreme JavaScript components.

Reviewer: ```suggestion
The DevExpress MCP server connects MCP-compatible AI tools with the DevExpress documentation library, including DevExtreme documentation. The MCP server ensures AI tools (such as GitHub Copilot) have direct access to the latest information on DevExtreme JavaScript components.
```

> [note] If you want reusable, task-focused prompts for AI coding assistants, see [DevExpress AI Skills](/concepts/90%20AI%20Features/30%20DevExpress%20AI%20Skills.md '/Documentation/Guide/AI_Features/DevExpress_AI_Skills/').

Reviewer: Let's stick to the rules as good as we can)

```suggestion
[note] If you want reusable, task-focused prompts for AI coding assistants, refer to the following help topic: [DevExpress AI Skills](/concepts/90%20AI%20Features/30%20DevExpress%20AI%20Skills.md '/Documentation/Guide/AI_Features/DevExpress_AI_Skills/').
```

> The DevExpress MCP server connects MCP-compatible AI tools with the DevExpress documentation library, including DevExtreme documentation. The MCP server ensures AI tools (such as GitHub Copilot) have direct access to the latest information on DevExtreme JavaScript components.

Reviewer 2: ```suggestion
The DevExpress MCP Server connects MCP-compatible AI tools with the DevExpress documentation library, including DevExtreme documentation. The MCP Server ensures AI tools (such as GitHub Copilot) have direct access to the latest information on DevExtreme JavaScript components.
```
